DFC.ServiceTaxonomy.GraphVisualiser/Services/Neo4JToOwlGeneratorService.cs

```diff
@@ -47,6 +47,8 @@ private void TransformData(IEnumerable<INode> nodes, string prefLabel)
                                   Label = GetPropertyValue(a, new[] { prefLabel, "Description", "FurtherInfo" }),
                                   Comment = GetPropertyValue(a, new[] { "Description" }),
                                   StaxProperties = a.Properties.Where(p => p.Key != prefLabel).Select(p => $"{p.Key}:{p.Value}").ToList(),
+                                  ContentType = a.Labels[1]
```

Comment:
The content type is already in the model as Type (see line 46) (and you can't assume the type is in the 2nd position).